apps/juxtaposition-ui/src/config.ts

```diff
@@ -8,6 +8,7 @@ const schema = z.object({
 	logSensitive: zodCoercedBoolean().default(false),
 	httpPort: z.coerce.number().default(8080),
 	httpCookieDomain: z.string().default('.pretendo.network'),
+	httpTrustProxy: z.union([zodStrictBoolean(), z.coerce.number(), z.string()]).default(false),
```

Comment:
The new config option supports a lot of cases - a lot more complicated then a boolean.

I think it's wise to add a comment explaining the basic inputs (cidr & boolean) and a link to https://expressjs.com/en/guide/behind-proxies.html for details.

(same for miiverse-api)